### PR TITLE
add `include_user` params in /messages/cursor` endpoint

### PR DESCRIPTION
**Pull request by notmd**
I need this to display user information in message table

### CODE REVIEW

File: backend/oasst_backend/prompt_repository.py [hunk 1]
```diff
@@ -888,14 +888,27 @@ def fetch_message_with_max_children(self, message: Message | UUID) -> tuple[Mess
         max_message = max(tree, key=lambda m: m.children_count)
         return max_message, [m for m in tree if m.parent_id == max_message.id]
 
-    def _add_user_emojis_all(self, qry: Query) -> list[Message]:
+    def _add_user_emojis_all(self, qry: Query, include_user=False) -> list[Message]:
```
Comment (andreaskoepf) [inline backend/oasst_backend/prompt_repository.py:891]: Could you add python type annotation?

File: backend/oasst_backend/models/message.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
 import sqlalchemy as sa
 import sqlalchemy.dialects.postgresql as pg
 from oasst_backend.models.db_payload import MessagePayload
+from oasst_backend.models.user import User
```
Comment (andreaskoepf) [inline backend/oasst_backend/models/message.py:9]: can also be imported from modes (omitting .user)

File: backend/oasst_backend/prompt_repository.py [hunk 3]
```diff
@@ -888,14 +888,27 @@ def fetch_message_with_max_children(self, message: Message | UUID) -> tuple[Mess
         max_message = max(tree, key=lambda m: m.children_count)
         return max_message, [m for m in tree if m.parent_id == max_message.id]
 
-    def _add_user_emojis_all(self, qry: Query) -> list[Message]:
```
Comment (andreaskoepf) [inline backend/oasst_backend/prompt_repository.py:891]: `_add_user_emojis_all` is a function that is used in several places as a wrapper around Message queries .. IMO this is a bit hacky to add an `include_user` param which is only needed at one place.

**Review by andreaskoepf — APPROVED** [review]

File: backend/oasst_backend/prompt_repository.py (hunk 1 above)
Comment (notmd) [reply backend/oasst_backend/prompt_repository.py]: Done

File: backend/oasst_backend/models/message.py (hunk 2 above)
Comment (notmd) [reply backend/oasst_backend/models/message.py:9]: Unfortunately, it causes circular import error
image

Comment (andreaskoepf) [reply backend/oasst_backend/models/message.py:9]: ah, ok, we are in the Message model file ;-) ..

**Pull request merged**